plugins/inputs/couchbase/couchbase.go

```diff
@@ -55,6 +58,21 @@ func (r *Couchbase) Gather(acc telegraf.Accumulator) error {
 	return nil
 }
 
+// sanitizeURI by removing information about user and/or password from string
+// it also removes schema name from URI
+func sanitizeURI(uri string) (result string, err error) {
+
+	re, err := regexp.Compile("(\\S+:\\/\\/)?(\\S+\\:\\S+@)")
```

Comment:
Move to package scope and use MustCompile, this way it only will be compiled once.  Personally I would use backtick strings so you don't need to double escape, also you don't need to escape `/` or `:`.  This would leave you with
```
`(\S+://)?(\S+\:\S+@)`
```

This is a nitpick, but it would be nicer if there was less vertical whitespace in this function.